info.toml

```diff
@@ -771,7 +771,7 @@ case is a vector of integers and the failure case is a DivisionError.
 
 The list_of_results function needs to return a vector of results.
 
-See https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect for how 
+See https://doc.rust-lang.org/std/iter/trait.Iterator.html#method.collect for how
```

Comment:
Looks like some extraneous whitespace was removed when I edited the file on github.dev. I can remove the white space change here and below if you want to keep the PR to just the line number  change.